ads/google/doubleclick.js

```diff
@@ -65,15 +65,30 @@ export function doubleclick(global, data) {
     transform: 'translate(-50%, -50%)',
   });
 
-  if (data.useSameDomainRenderingUntilDeprecated != undefined ||
-      data.multiSize) {
-    doubleClickWithGpt(global, data, GladeExperiment.GLADE_OPT_OUT);
+  let url = 'https://www.googletagservices.com/tag/js/';
+  let sf = false;
+  const expIds = data['experimentId'];
```

Comment:
I could well be mistaken but I think it needs to be outside the if because I pass the url to doubleclickwithgpt in a different else block later?